### ISSUE TITLE
Template selector is not keyboard accessible

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/tables#248]
The tiles for table templates in the new table dialog are currently not accessible by keyboard navigation.

In addition we would also need to indicate to screen readers which the currently selected element is.

**Comment by juliushaertl** [comment]
Retested now and the entries are keyboard navigatable but selecting them is not possible -> follow up fix in #505
